code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -250,6 +250,23 @@
 	//min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	flags = NODROP
 
+/obj/item/clothing/suit/hooded/bloated_human	//OH MY GOD WHAT HAVE YOU DONE!?!?!?
+	name = "bloated human suit"

```

Comment:
totally make its name the same as the lings' (bloated flesh mass right?) for maximum honks as clown.